.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# create folders
+mkdir -p /workspaces/InvenTree/dev/{commandhistory,plugins}
+cd /workspaces/InvenTree
+
+# create venv
+python3 -m venv dev/venv
+. dev/venv/bin/activate
+
+# setup inventree server
+pip install invoke
+inv update
+inv setup-test --ignore-update --path dev/demo-dataset
```

Comment:
You're right that this "setup-test" should not be performed "every time" the container is built. We should suggest it as a one-time step after setup.

Can you revert this to the "setup-dev" command?